### PR TITLE
test(idd): cover untested authoring-label-guard exports

### LINKED ISSUE #1088: test(idd): cover the two untested authoring-label-guard exports
## Background

Parent roadmap: #1076

`src/scripts/authoring-label-guard.mts` exports `resolveAuthoringGuardPolicy`
and `formatElapsedDuration`, but a repo-wide search finds no test references to
either — the dedicated `tests/authoring-label-guard.test.mts` imports only
`buildAuthoringLabelWarning` + `findLatestLabeledAt`, and its single warning
case exercises only the `timestamp_unavailable` path, so `formatElapsedDuration`
is never reached even indirectly. Both are pure/deterministic and currently
uncovered.

## Proposed change

Add regression tests (node:test, matching the existing suite) for:

- `formatElapsedDuration`: d/h/m composition, zero-unit omission, and the
  sub-minute / negative clamp to `0m`.
- `resolveAuthoringGuardPolicy`: config + override resolution, including an
  invalid ISO-8601 `authoringStaleAge` falling back to the default stale age
  (note the fallback is realized via `normalizePolicyConfig`'s `parseDuration`
  in `policy-helpers.mts`, so drive it through the normalize→resolve chain).
- Boundary cases via `buildAuthoringLabelWarning`: stale exactly at threshold,
  and `timestamp_unavailable` when no matching event exists.

## Acceptance criteria

- New tests cover `formatElapsedDuration` (d/h/m, zero-unit omission,
  sub-minute/negative → `0m`) and `resolveAuthoringGuardPolicy`
  (invalid-duration fallback) and pass.
- The stale-exactly-at-threshold boundary is asserted.
- `pnpm test`, `pnpm run lint:minimum` pass. (Tests are not emitted; no `.mjs`
  rebuild needed for a test-only change.)

## Candidate files

- `tests/authoring-label-guard.test.mts`
- reference: `src/scripts/authoring-label-guard.mts`,
  `src/scripts/policy-helpers.mts` (`normalizePolicyConfig`, `parseDuration`)

---

_Autopilot suitability: 4 / 5 -- higher is more autopilot-suitable; below the configured floor is human-oriented._

_Effort: S -- author-estimated size; a soft autopilot selection tie-breaker only._

### PR DESCRIPTION
**Pull request by kurone-kito**
## Summary

Adds `node:test` regression coverage for the two previously-uncovered
exports of `src/scripts/authoring-label-guard.mts`:

- `formatElapsedDuration` — d/h/m composition, zero-unit omission, the
  sub-minute / negative clamp to `0m`, and a fractional-ms floor.
- `resolveAuthoringGuardPolicy` — default and label/stale-age override
  resolution, plus an invalid ISO-8601 `authoringStaleAge` falling back
  to the default age. The fallback is exercised through the real
  `normalizePolicyConfig` (`parseDuration`) -> resolve chain rather than a
  pre-normalized input, per the issue.

Also asserts the `buildAuthoringLabelWarning` stale boundary — exactly at
the threshold is `stale` (and its message renders `formatElapsedDuration`),
one millisecond under is silent — and `timestamp_unavailable` when no
labeled event matches.

## Notes

Test-only change: tests are not emitted, so no `.mjs` rebuild is needed and
`scripts` / `bin` are unchanged. The new assertions for the default and
invalid-duration paths track `POLICY_DEFAULTS` (via `parseIsoDurationToMs`),
while the override case independently pins `PT2H` to keep those assertions
non-tautological. `pnpm run lint:minimum` and `pnpm test` pass.

Closes #1088

### CODE REVIEW

**Comment by kurone-kito** [comment]
**Rejected** — chatgpt-codex-connector[bot] did not review HEAD ac3dfb9cde68156de13e0bd0c86a6164b8aabadc (Codex usage limits for code reviews reached); this is not a completed review

**Comment by kurone-kito** [comment]
**Rejected** — coderabbitai[bot] did not review HEAD ac3dfb9cde68156de13e0bd0c86a6164b8aabadc (review limit reached / rate limited); this is not a completed review

File: tests/authoring-label-guard.test.mts [hunk 1]
```diff
@@ -99,3 +110,125 @@ test('buildAuthoringLabelWarning treats implicit events as timestamp unavailable
...
+  assert.equal(formatElapsedDuration(2 * HOUR_MS), '2h');
+});
+
+test('formatElapsedDuration clamps sub-minute and negative inputs to 0m', () => {
+  assert.equal(formatElapsedDuration(30 * 1000), '0m'); // 30s, under a minute
+  assert.equal(formatElapsedDuration(0), '0m');
+  assert.equal(formatElapsedDuration(-5 * MINUTE_MS), '0m'); // negative clamps
+});
+
+test('formatElapsedDuration floors fractional milliseconds down to a minute', () => {
```
[the remark this answers is not in the document]
Comment (kurone-kito) [reply tests/authoring-label-guard.test.mts:193]: **Rejected** — advisory cosmetic nit on a test name; no behavior or coverage change. The assertion (input MINUTE_MS + 0.9 rendering "1m") already documents the integer-millisecond floor before unit composition, so the name is acceptable. Declining the rename-only change to keep this test-only PR converged.

**Comment by kurone-kito** [comment]
_claude-83ac63: review triage snapshot — IDD automation marker. Do not edit._

**Comment by kurone-kito** [comment]
**F4 Cleanup Evidence**

| Field              | Value   |
| ------------------ | ------- |
| Status             | applied |
| Applied            | 2       |
| Failed             | 0       |
| Skipped            | 3       |
| Permission-blocked | 0       |
| Notes              | Agent-side cleanup: minimized the review-watermark marker (OUTDATED) and the resolved Copilot review parent (RESOLVED). 1 Copilot review comment was already minimized by the concurrent post-merge-cleanup workflow; the 2 advisory-bot rate-limit comments are retained with their disposition replies. |
